docs/reference/ccr/index.asciidoc

```diff
@@ -8,6 +8,8 @@
 
 beta[]
 
+WARNING: <<index-lifecycle-management,{ILM}>> is not supported by indices being managed by {ccr}
```

Comment:
On the leader index side, the index is not really "managed" by CCR, per se. An alternative wording might be to say leader indices or following indices, and link to the CCR documentation that defines these terms?

Reply:
sure thing, I agree. thanks for the recommendation.
I wasn't sure how detailed to go.

I am also confused as to why this isn't being rendered in the docs, while the warning on the ILM side is